src/tokens/solana.tokenlist.json

```diff
@@ -137401,4 +137417,4 @@
     "minor": 3,
     "patch": 3
   }
-}
\ No newline at end of file
+}
```

Comment:
wtf

Reply:
" dontfix extra line
set nofixendofline
 in the config

 and last char in vim is }, 
i dont understand what else can be done

Reply:
" dontfix extra line
set nofixendofline

doesnt seem to help in vim config to make my line the same No newline at the end of file